docs/docs/setting-up/jobs/job-specification/task.md

```diff
@@ -8,12 +8,12 @@ A `Task` signifies a distinct unit of work within the broader context of a `Job`
 
 ## `Task` Parameters
 - **Name** `(string : <required>)`: A unique identifier representing the name of the task.
-- **Engine** `(`[`SpecConfig`](./spec-config)` : required)`: Configures the execution engine for the task, such as [Docker](../other-specifications/engines/docker) or [WebAssembly](../other-specifications/engines/wasm).
-- **Publisher** `(`[`SpecConfig`](./spec-config)` : optional)`: Specifies where the results of the task should be published, such as [S3](../other-specifications/publishers/s3) and [IPFS](../other-specifications/publishers/ipfs) publishers. Only applicable for tasks of type `batch` and `ops`.
+- **Engine** <code>(<a href="./spec-config">SpecConfig</a>: \<required>\)</code>: Configures the execution engine for the task, such as [Docker](../../other-specifications/engines/docker.md) or [WebAssembly](../../other-specifications/engines/docker.md).
```

Comment:
The link for WebAssembly is incorrect, it should be ../other-specifications/engines/wasm

Reply:
fixed